### DISCUSSION TITLE
Alist webdav挂载后访问视频报无权限访问

### DISCUSSION THREAD

**Discussion by Ai-Data-Man** [AlistGo/alist#4774] [Q&A]
Alist Version / Alist 版本：
v3.21.0
Driver used / 使用的存储驱动:
百度网盘
本地操作系统：
<img width="638" alt="quicker_d5efcc73-65af-4217-9763-40fcdfdf887b" src="https://github.com/alist-org/alist/assets/26343431/322a4279-da67-4369-8e94-b15b82d81d75">
Alist安装方式：
安装在本地操作系统
Alist配置：
<img width="1032" alt="Untitled" src="https://github.com/alist-org/alist/assets/26343431/c238768e-e72e-48b2-aec3-309d65a8837e">

问题描述：
通过MountainDuck或是Win11自带的映射地址到网络驱动器功能将Alist添加的百度网盘目录映射为本地网络驱动器时，访问其中的视频文件（即使该视频文件仅1MB多）将会出现如下报错：
<img width="898" alt="image" src="https://github.com/alist-org/alist/assets/26343431/28f28130-10c0-481a-86fc-db9b4db2f8fa">
但是如果是访问同目录下的一个小txt文件，没有任何问题：
<img width="940" alt="image" src="https://github.com/alist-org/alist/assets/26343431/b3cccca2-fc6a-49df-8900-a28bde8a6efc">
并且，可以在Alist UI播放任何视频文件：
<img width="741" alt="quicker_7383de49-a09c-40fb-841d-c14aaf1240a9" src="https://github.com/alist-org/alist/assets/26343431/4b18824d-5196-425d-8468-c494c849a949">

**Comment by LegendAdmin** [comment]
我也不知道，但我见网上的评论时间来看是个老问题了
我的解决办法就是不行就算了，用RaiDrive挂载WebDav
不过要是能解决就更好了

**Reply by Ai-Data-Man** [reply]
我用RailDrive挂载也不行。。。

**Reply by LegendAdmin** [reply]
WebDav策略改一下看看，我记得有群友遇到过你这种情况，改了WebDav策略就好了

**Reply by qi-gu** [reply]
有用，感谢……但我不理解的是之前用了三周一直没问题，突然有一天就打不开了……改了之后确实又能顺畅播放
